### DISCUSSION TITLE
Minio leak memory

### DISCUSSION THREAD

**Discussion by sinh-tran-ts** [minio/minio#18379] [Q&A] [answered]
Hello everyone,
We have just upgraded Minio from RELEASE.2022-10-24T18-35-07Z to RELEASE.2022-06-20T23-13-45Z. However, we are currently experiencing a memory leak issue.
Please assist me in finding a solution. I have gathered some information.
Thanks!

[profile-127.0.0.1-9000-mem.txt](https://github.com/minio/minio/files/13238458/profile-127.0.0.1-9000-mem.txt)

[profile-127.0.0.1-9000-goroutines.txt](https://github.com/minio/minio/files/13238461/profile-127.0.0.1-9000-goroutines.txt)
[profile-127.0.0.1-9000-goroutines-before.txt](https://github.com/minio/minio/files/13238462/profile-127.0.0.1-9000-goroutines-before.txt)
[profile-127.0.0.1-9000-goroutines-before-debug=2.txt](https://github.com/minio/minio/files/13238464/profile-127.0.0.1-9000-goroutines-before-debug.2.txt)
![8512b2c6-a53b-4c8d-976a-9738f7ed5a16](https://github.com/minio/minio/assets/104056516/366750b4-c28b-4500-9b30-1b5472718534)
<img width="1129" alt="Screenshot 2023-11-02 at 15 32 53" src="https://github.com/minio/minio/assets/104056516/6d30bfcb-0262-4571-af80-1a595073c1b0">
<img width="1427" alt="Screenshot 2023-11-02 at 15 46 30" src="https://github.com/minio/minio/assets/104056516/555021af-4e29-4137-ab24-6cb28a575a3a">
<img width="1400" alt="Screenshot 2023-11-01 at 11 51 30" src="https://github.com/minio/minio/assets/104056516/2ef428e5-c5b4-4d1d-a8b7-51517446dbf8">

**Comment by klauspost** [comment, marked as answer]
Upgrade to latest. We've fixed at least one issue related to this since then.
